tests/beaker/tests/create_lv_with_param_stripesize.rb

```diff
@@ -0,0 +1,52 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96587 - create logical volume with parameter 'stripes'"
+test_name "FM-4579 - C96633 - create logical volume with more than 1 stripes"
+test_name "FM-4579 - C96589 - create logical volume with parameter 'volume_group'"
```

Comment:
Avoid having multiple tests in a single test file. If one test can represent a class of tests then you should only use that single test. If the tests are unique and valuable then the tests should be in separate files.